scripts/linking.py

```diff
@@ -11,7 +11,7 @@
 import datetime
 from collections import defaultdict
 from tqdm import tqdm
-
+import jsonlines
```

Comment:
Can we not use 3rd party libraries for this, given you can easily achieve the same thing with `json`.

Reply:
done